soccer/gameplay/skills/_kick.py

```diff
@@ -24,6 +24,8 @@ def __init__(self):
 
         self.shot_obstacle_ignoring_robots = []
 
+        self.aim_params = {}
```

Comment:
I made some changes to _kick to allow `line_kick` to work with coordinated pass. Its a little bit hacky, but it should allow anything that subclasses kick to be used in coordinated_pass.